src/views/primary-auth/PrimaryAuthForm.js

```diff
@@ -191,6 +191,22 @@ define([
       if (this.settings.get('features.trackTypingPattern')) {
         TypingUtil.track('okta-signin-username');
       }
-    }
+    },
+
+    // TODO: support a11y attrs in Courage - OKTA-279025
+    render: function () {
+      Okta.Form.prototype.render.call(this);
+
+      this.$el.attr('novalidate', 'novalidate');
```

Comment:
You can define attributes for the view by adding an `attributes` property to the Form definition: https://backbonejs.org/#View-attributes Here's an example: https://github.com/okta/okta-signin-widget/blob/master/src/AccountUnlockedController.js#L37

Reply:
done